packages/govern-subgraph/src/GovernQueue.ts

```diff
@@ -147,35 +145,6 @@ export function handleConfigured(event: ConfiguredEvent): void {
   queue.save()
 }
 
-// IArbitrable Events
-
-export function handleEvidenceSubmitted(event: EvidenceSubmittedEvent): void {
-  let governQueue = GovernQueueContract.bind(event.address)
-  let containerHash = governQueue.disputeItemCache(
-    event.params.arbitrator,
-    event.params.disputeId
-  )
-  let container = loadOrCreateContainer(containerHash)
-
-  handleContainerEventSubmitEvidence(container, event)
-}
-
-export function handleRuled(event: RuledEvent): void {
```

Comment:
The ``Ruled`` event is still existing on the ``GovernQueue`` [here](https://github.com/aragon/govern/blob/master/packages/govern-core/contracts/pipelines/GovernQueue.sol#L213). 

Was there another reason why this got removed?

Reply:
`HandleRuled` event seems to be changed.  In the subgraph mapping, this is what we had:

```
let containerHash = governQueue.disputeItemCache(
    event.params.arbitrator,
    event.params.disputeId
  )
```

Since `disputeItemCache` was changed too, meaning that the first key we should pass is `containerHash`, it's impossible to get containerHash from the above code. Which means `Ruled` event doesn't help at all.

Reply:
Thanks for the discussion on Discord. Anything looks good 👍